_posts/2019-01-XX-enabling-cpb-on-pcengines-apu2.md

```diff
@@ -0,0 +1,416 @@
+---
+post_title: Enabling Core Performance Boost on PC Engines apu2
+author: Michał Żygowski
+layout: post
+published: false
+post_date: 2019-01-DD HH:MM:SS
+
+tags:
+    - PC Engines
+    - apu2
+    - boost
+    - performance
+    - coreboot
+categories:
+    - Firmware
+
+---
+
+# Pushing hardware to its limits
+
+In the epoch of efficient and fast processors, performance becomes one of the
+most crucial aspects when choosing and working with hardware. We want our
+computers to execute their tasks with possibly highest speeds. But what really
+influences the performance of our platforms? It's the processor's manufacturer
+design one may say. In this post, I will show You how firmware may boost Your
+silicon to higher performance level. On the example of PC Engines apu2c4
+platform, I will present Core Performance Boost feature.
+
+## Core Performance Boost
+
+![BOOST](https://3mdeb.com/wp-content/uploads/2019/01/boost.jpeg)
```

Comment:
@pietrushnic just searched in google. Licence unknown. Hard to get fancy graphics for "boost" keyword, will have to replace it.

Reply:
@miczyg1 never use random graphics from internet. If you have to please use google search images and mark "free for commercial use". Otherwise please use pictures, screenshots and diagrams made internally. We will build marketing materials library over time.

Reply:
Found a CC0 graphics and edited it a little bit to my needs.